lumicks/pylake/population/dwelltime.py

```diff
@@ -480,3 +480,55 @@ def _exponential_mle_optimize(
 
     # output parameters as [amplitudes, lifetimes], -log_likelihood
     return result.x, -result.fun
+
+
+def _dwellcounts_from_statepath(statepath, exclude_ambiguous_dwells=True):
+    """Calculate the dwell counts and slicing indices for all states in a state path trajectory.
+
+    Note: the counts are the number of frames or time points. To convert to proper
+    dwelltimes, multiply the counts by the time step between points.
+
+    Parameters
+    ----------
+    statepath : numpy.typing.ArrayLike
```

Comment:
I think here just using `array_like` is fine? The `numpy.typing` stuff was for the type annotations.